node/src/actors/chain_manager/mod.rs

```diff
@@ -987,6 +995,13 @@ fn update_reputation(
     // Penalize liars and accumulate the reputation
     // The penalization depends on the number of lies from the last epoch
     let liars_and_penalize_function = liars.iter().map(|(pkh, num_lies)| {
+        if own_pkh == *pkh {
+            log::warn!(
+                "Your reputation has been slashed. Your lies number was {}",
```

Comment:
```suggestion
                "Your reputation score has been slashed because you witnessed {} times out of consensus",
```

Reply:
Can we know how much did we lost?

Reply:
I agree, also to be consistent with the case when reputation is gained.